vtds_cluster_kvm/private/scripts/disk_builder.py

```diff
@@ -94,18 +94,7 @@ def __init__(self, config, node_class, node_instance, root_passwd):
         self.host_dir = path_join(self.nodeclass_dir, node_name)
         makedirs(self.host_dir, mode=0o755, exist_ok=True)
         virtual_machine = node_class.get('virtual_machine', {})
```

Comment:
REVIEWERS: network booted nodes are not required to have boot disks, so don't error out when the boot disk is missing.